package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.1.1",
   "private": true,
   "engines": {
-    "node": ">=8.9.4 <10.0.0"
+    "node": ">=8.9.4"
```

Comment:
should this be ` >=10.1.0`?

Reply:
nah need jenkins to be upgraded for that, which requires all the teams to update

Reply:
ok